jgrapht-core/src/main/java/org/jgrapht/GraphTests.java

```diff
@@ -496,6 +497,32 @@
         return new ChordalityInspector<>(graph).isChordal();
     }
 
+    /**
+     * Checks whether a graph is <a href="http://www.graphclasses.org/classes/gc_14.html">weakly chordal</a>.
+     * <p>
+     * The following definitions are equivalent:
+     * <ol>
+     * <li> A graph is weakly chordal if it is (<a href="http://mathworld.wolfram.com/GraphAntihole.html">
```

Comment:
Let's make the description identical to the one found in WeakChordalityInspector